treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -42,6 +44,8 @@
 
       Details of the alert can be found in the [alert summary]({{ alertHref }}), including links to graphs and comparisons for each of the affected tests. Please follow our [guide to handling regression bugs](https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs) and **let us know your plans within 3 business days, or the offending patch(es) will be backed out** in accordance with our [regression policy](https://www.mozilla.org/en-US/about/governance/policies/regressions/).
 
+      If you need the profiling jobs you can trigger them yourself from treeherder job view or ask a sheriff to do that for you.
```

Comment:
please don't include this for build_metrics

Reply:
build_metrics is framework 2, this line here is for framework 4 which is awsy.